src/components/spreadsheet/spreadsheet.ts

```diff
@@ -155,7 +153,6 @@ export class Spreadsheet extends Component<SpreadsheetProps, SpreadsheetChildEnv
 
   setup() {
     this.props.exposeSpreadsheet?.(this);
-    this.model = this.props.model;
```

Comment:
why ?
the lines are now longer to write and we gain nothing ?

Reply:
`this.model` and `this.props.model` can be desynchronized (there is no willUpdateProps for exemple). It's the same reason why we remove `this.getters` on the components.

Reply:
I faced an issue where I tried to mock something on the model in a test, and I lost 30 minutes to discover this --'

Reply:
If you prefer, I can do:
```js
get model() { return this.props.model } 
```
but as the model does not belong to `Spreadsheet`, I do not want to store it on `this`